composer.json

```
@@ -18,7 +18,7 @@
     ],
 
     "require": {
-        "php":                          "^7.1.3",
+        "php":                          "^7.1.3|^8.0",
```

```suggestion
        "php":                          ">=7.1.3",
```
This essentially does the same thing, except is doesn't put an upper version limit on PHP. It's also what Symfony has adopted for PHP 8 support